.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Setup Bun
+      uses: oven-sh/setup-bun@v1
+      with:
+        bun-version: '1.0.30'
+        cache: true
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)